tests/onboarding/test_password_strength.py

```diff
@@ -0,0 +1,50 @@
+import allure
+import pytest
+from allure_commons._allure import step
+
+import configs.system
+import constants
+from constants.onboarding import very_weak_lower_elements, very_weak_upper_elements, \
+    very_weak_numbers_elements, very_weak_symbols_elements, weak_elements, so_so_elements, good_elements, great_elements
+from gui.components.onboarding.before_started_popup import BeforeStartedPopUp
+from gui.screens.onboarding import AllowNotificationsView, WelcomeToStatusView, KeysView
+
+
+@pytest.fixture
+def keys_screen(main_window) -> KeysView:
+    with step('Open Generate new keys view'):
+        if configs.system.IS_MAC:
+            AllowNotificationsView().wait_until_appears().allow()
+        BeforeStartedPopUp().get_started()
+        wellcome_screen = WelcomeToStatusView().wait_until_appears()
+        return wellcome_screen.get_keys()
+
+
+@allure.testcase('https://ethstatus.testrail.net/index.php?/cases/view/702989',
+                 'Strength of the password')
+@pytest.mark.case(702989)
+@pytest.mark.parametrize('user_account', [constants.user.user_account_one])
+@pytest.mark.parametrize('password, password_strength_elements', [
+    pytest.param('abc', very_weak_lower_elements),
+    pytest.param('ABC', very_weak_upper_elements),
+    pytest.param('123', very_weak_numbers_elements),
+    pytest.param('+_!', very_weak_symbols_elements),
+    pytest.param('+1_3!48', weak_elements),
+    pytest.param('+1_3!48a', so_so_elements),
+    pytest.param('+1_3!48aT', good_elements),
+    pytest.param('+1_3!48aTq', great_elements)
+])
+def test_password_strength(keys_screen, user_account, password: str, password_strength_elements):
+    with step('Input correct user name'):
+        profile_view = keys_screen.generate_new_keys()
+        profile_view.set_display_name(user_account.name)
+
+    with step('Input password'):
+        details_view = profile_view.next()
+        create_password_view = details_view.next()
+        create_password_view.set_password_in_first_field(password)
+
+    with step('Verify that correct strength indicator color, text and green messages appear'):
+        assert create_password_view.strength_indicator_color == password_strength_elements[1]
+        assert create_password_view.strength_indicator_text == password_strength_elements[0]
+        assert sorted(create_password_view.green_indicator_messages) == sorted(password_strength_elements[2])
```

Comment:
Can we add one more step that checks the successful login (after all the invalid entries)? Just to make sure user still can get in after the validation form errors ( it was not present in old test, so just a suggestion to make the test more like a functional test)

Reply:
Ok. Great idea. I will add it too. And then will update test case as well

Reply:
Done. The only thing is that with logging in this test run takes 6 minutes locally